docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -487,10 +487,10 @@ When fields, which are lists (e.g. `scopesSupported`, `responseTypesSupported`),
 
 The client secret can be input directly, or for added security it can be aliased using any of the following features:
 
-- xref:/documentation/payara-server/password-aliases/README.adoc[Password Aliases]
-- xref:/documentation/payara-server/server-configuration/var-substitution/README.adoc[Environment Variables / System Properties]
 - xref:/documentation/microprofile/config/README.adoc[Config API]
 
+NOTE: Environment Variables and System Properties can be used indirectly through MicroProfile Config.
```

Comment:
This is what the previous text said already?

Reply:
Environment Variables and System Properties cannot be used via variable substitution, if a user wants to use them, they will have to use them indirectly through MicroProfile Config